### PR TITLE
Feature/scala

### PR DESCRIPTION
**Pull request by ghost**
Eerste opzet van de Scala pagina. Er kan nog veel meer bij, maar het leek me eerst goed om eerst maar wat neer te zetten en vanuit daar verder te bouwen

@wvandrunen, @q42jaap, @arjenvanderende, @qamil, @jaapmengers, @lvdo, @qristin, @chees, @Primigenus jullie feedback of eventuele toevoegingen zijn welkom!

### CODE REVIEW

**Comment by Primigenus** [comment]
Looks good!

I think we should have more content on the page before we actually add it to the site. Right now it's just a description of what Scala is and it doesn't really tie back into how we use it. Which projects or libraries have we created with Scala that you'd like to see here?

Also, I would prefer an image other than just the Scala logo. Any ideas?

**Comment by arjenvanderende** [comment]
It's a little big, but I like this poster :smile: 

The functional way is the right way

File: views/en/page/scala.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      <h1 style="display: none;">Scala at Q42</h1>
+    </header>
+
+    <section class="block-large">
+      <div class="intro">
+        <a class="visual caption">
+          <img data-src="http://static.q42.nl/images/scala/logo.png" alt="Scala at Q42">
+        </a>
+        <p>
+          Scala is an object-functional programming language for general software applications. Scala has full support for functional programming and a very strong static type system. This allows programs written in Scala to be very concise and thus smaller in size than other general-purpose programming languages.Java libraries may be used directly in Scala code and vice versa. This means you can use a lot of the JVM libraries published in the past decade! But it doesn't mean we use new libraries like; Akka, Scalaz, Play!, etc on the contrary!
```
Comment (chees) [inline views/en/page/scala.html]: This last sentence seems weird. Is that really what you mean?

**Comment by q42mark** [comment]
@arjenvanderende Haha, well you can resize if you want to ofcourse. Anything else ?

File: views/en/page/scala.html (hunk 1 above)
Comment (q42mark) [inline views/en/page/scala.html]: You are right, i'll correct this! Anything else you want to add?

**Comment by q42mark** [comment]
@Primigenus Added the project Malmberg BAPS. Would love to hear you review :) Screenshot will be replaced with something better, any suggestions?

@arjenvanderende Would you like to write something about Philips Hue?

**Comment by arjenvanderende** [comment]
@q42mark I think it's too premature to mention something about Philips Hue, because no Scala components are running in production yet. I'll try to remember updating this page when we do, though :smile:

**Pull request closed without merging**